### PR TITLE
Fix Tweets in `sublime` extension, and change API params

### PR DESCRIPTION
**Pull request by phgn0**
## Description

Fix the display of Tweets inside the `Sublime` extension (before it showed an empty body):

Screenshot 2024-10-09 at 12 32 29 PM

We also changed the URL of the API used by the `Sublime` extension, and renamed one parameter. This is backwards compatible.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/sublime/scripts/patch-prod.js
Comment (phgn0) [inline extensions/sublime/scripts/patch-prod.js:1]: Remove some private development scripts.

File: extensions/sublime/src/utils/api.ts [hunk 1]
```diff
@@ -187,7 +187,7 @@ export async function saveLink(
             mark_as_favorite: isFavorite,
             mark_as_private: isPrivate,
             contributions: note ? [note] : [],
-            connections: connectedCollections.map((c) => c.uuid),
+            connect: connectedCollections.map((c) => c.uuid),
```
Comment (phgn0) [inline extensions/sublime/src/utils/api.ts:190]: A backwards-compatible API change.

File: extensions/sublime/src/views/list.tsx [hunk 2]
```diff
@@ -38,18 +38,20 @@ export default function CardsList({
...
                     icon={getEntityIcon(item)}
                     detail={
                         <List.Item.Detail
                             markdown={
-                                item.thumbnail
-                                    ? `<img alt="Card image" src="${item.thumbnail}" />`
-                                    : (item.name ? `# ${item.name}\n` : "") + item.markdown
-
-                                // `> ${JSON.stringify(item, null, 2)}`
+                                (item.name ? `# ${item.name}\n` : "") +
```
Comment (phgn0) [inline extensions/sublime/src/views/list.tsx:50]: Show all available properties.

**Comment by phgn0** [comment]
> Is this ready for a review? 🙂

Yes it's ready! I added the inline comments only to make the review easier! :)

**Comment by pernielsentikaer** [comment]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**